server/lib/sanitize-html.ts

```diff
@@ -137,8 +137,17 @@ export const generateSummaryForHTML = (content, maxLength = 255) => {
     .replace(/\s+/g, ' ');
 
   // Truncate
-  const truncated = truncate(trimmed, { length: maxLength });
+  const truncated = truncate(trimmed, { length: maxLength, omission: ''});
 
   // Second sanitize pass: an additional precaution in case someones finds a way to play with the trimmed version
-  return sanitizeHTML(truncated, optsSanitizeSummary);
+  const secondSanitized = sanitizeHTML(truncated, optsSanitizeSummary);
+
+  const isTruncated = trimmed.length > maxLength;
+
+  // Check to see if the second sanitization cuts a html tag in the middle
+  if (isTruncated && secondSanitized.slice(secondSanitized.length - 3) !== '...') {
```

Comment:
Because we now have `omission: ''`, is this `secondSanitized.slice(secondSanitized.length - 3) !== '...'` check still necessary?

Reply:
@Betree : Ops, you are correct, that's not needed anymore. I've removed it. Thanks a bunch. :+1: